packages/data/README.md

```diff
@@ -70,130 +70,121 @@ yarn add @semaphore-protocol/data
 
 ## 📜 Usage
 
-For more information on the functions provided by `@semaphore-protocol/data`, please refer to the [TypeDoc documentation](https://js.semaphore.pse.dev/modules/_semaphore_protocol_data).
+For detailed information on the functions provided by `@semaphore-protocol/data`, please refer to the [TypeDoc documentation](https://js.semaphore.pse.dev/modules/_semaphore_protocol_data).
 
-\# **getSupportedNetworks**(): _string[]_
+### Creating and Managing Subgraphs
 
-```typescript
-const supportedNetworks = getSupportedNetworks()
-```
-
-\# **new SemaphoreSubgraph**(networkOrSubgraphURL: SupportedNetwork | ValueOf\<SupportedNetwork> | string = "sepolia"): _SemaphoreSubgraph_
+**Initialize a Semaphore Subgraph instance**
 
 ```typescript
 import { SemaphoreSubgraph } from "@semaphore-protocol/data"
 
+const supportedNetworks = getSupportedNetworks()
```

Comment:
We need to update this I think. That function has been moved to the `@semphore-protocol/utils` package. It's an object now. You can check that package to know more. There's no enum or something like `supportedNetworks.arbitrum` tho, so I'm not sure if we should use it here. A `"arbitrum"` string should be the right way, unless we don't add an enum to the utils package.